### PR TITLE
doc: storage: clarify that most docs are about custom volumes

### PR DESCRIPTION
**Pull request by ru-fu**
You cannot do the same things with an instance volume as you
can with custom volumes.

Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

File: doc/howto/storage_configure.md [hunk 1]
```diff
@@ -34,12 +34,12 @@ You can also edit the storage volume configuration by using the following comman
 
     lxc storage volume edit <pool_name> <volume_name>
 
-## Configure default values for storage volumes
+## Configure default values for custom storage volumes
```
Comment (tomponline) [inline doc/howto/storage_configure.md:37]: These settings also apply to non-custom volumes (when not overridden via the instance's config).

File: doc/howto/storage_configure.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ You can also edit the storage pool configuration by using the following command:
     lxc storage edit <pool_name>
 
 (storage-configure-volumes)=
-## Configure storage volumes
+## Configure custom storage volumes
```
Comment (tomponline) [inline doc/howto/storage_configure.md:23]: You can set some config keys on non-custom volumes such as `user.*` for example. So this section does still apply somewhat to non-custom volumes.

File: doc/howto/storage_backup.md [hunk 3]
```diff
@@ -79,27 +79,27 @@ See the {ref}`storage-drivers` documentation for more information about those co
 
 ### Restore a snapshot
```
Comment (tomponline) [inline doc/howto/storage_backup.md:80]: Restore a custom volume snapshot?

File: doc/howto/storage_backup.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ To delete a snapshot, use the following command:
 
 ### Schedule snapshots
```
Comment (tomponline) [inline doc/howto/storage_backup.md:64]: Custom volume scheduled snapshots?

File: doc/howto/storage_configure.md (hunk 2 above)
Comment (ru-fu) [reply doc/howto/storage_configure.md:23]: With the listed commands? I only get "storage pool volume not found" when I try to change the config for a container volume.

**Pull request merged**